src/partials/course.hbs

```diff
@@ -20,7 +20,7 @@
         <span class="price">{{#if price}}₹{{{price}}}{{/if}}</span>
         <p class="description">{{description.[0]}}</p>
         <p class="lead">{{{lectures}}}</p>
-        <a class="btn btn-white btn-filled btn-lg" href="{{#relativePath @file.path link}}{{/relativePath}}">{{#unless
+        <a class="btn btn-white btn-filled btn-lg" href="{{online-link}}">{{#unless
```

Comment:
Will break things for offline courses. Add a proper if check please.

Reply:
Hw so? The classroom courses still redirect to their relative paths. Please explain @witty123

Reply:
online-link won't exist for offline courses

Reply:
Made the changes! @witty123

Reply:
This will cause to have multiple register now buttons. Please check common stuff before submitting a PR.